### ISSUE TITLE
Multiclass probability labels

### ISSUE THREAD

**Issue by josemafuen** [keras-team/keras#12446]
I have a problem with seven classes and my training data is labeled with probability labels. That is to say, for example I could have one image that may belong to class 0 with 30% of probability and to class 3 with 70%, so its label would be [0.3, 0, 0, 0.7, 0, 0, 0]. 

The issue is that I have no clue of what loss measure should I use, because categorical_crossentropy should be used when each image belongs to a single class, and binary_crossentropy is usually used when each image belongs to one or more than one class (but for sure, not with a probability). For now I'm using binary_crossentropy because it seems more adequate.

Thank you for any help.

**Comment by czgdp1807** [comment]
I think `categorical_crossentropy` would be a more appropriate loss function for your task. In my point of view, `binary_crossentropy` means that the target label will be either 0 or 1, as the name binary suggests. However, `categorical_crossentropy` means that, the target label will be continuous values, which is your case. 

**References**
..[1] [Stackoverflow thread](https://stackoverflow.com/a/41913968)
..[2] [Loss functions-ML Cheatsheet](https://ml-cheatsheet.readthedocs.io/en/latest/loss_functions.html)
..[3] [Reddit thread](https://www.reddit.com/r/learnmachinelearning/comments/88g8zf/difference_between_binary_cross_entropy_and/dwl1vsb)

Please correct me if I am wrong since I am also a learner. :-)

**Comment by jvishnuvardhan** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.
To ask questions, please see the following resources :
[StackOverflow](http://stackoverflow.com/questions/tagged/keras)
[Keras Slack channel](https://keras-slack-autojoin.herokuapp.com)
[Keras.io support](https://keras.io/#support)
[Gitter](https://gitter.im/Keras-io/Lobby)
Thanks!
If you think we made a mistake, please open another issue and fill all the information asked by the template. Thanks!
